memcopy.h

```diff
@@ -266,6 +266,12 @@ static inline unsigned char *chunk_memcpy(unsigned char *out, unsigned char *fro
     return out;
 }
 
+#if defined(__arm__) || defined(__aarch64__) || defined(_M_ARM)
```

Comment:
I know that calling memset is better for x86_64, so it is better to change the ifdef like this:
```
#undef CALL_MEMSET
#if defined(__x86_64) || defined(__i386_)
# define CALL_MEMSET 1
#endif
```